### PR TITLE
docs: small fix: add local toc to nrfxlib RST pages.

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added local toc to nrfxlib RST pages.
Jira NCSDK-6534.

Signed-off-by: Francesco Servidio <francesco.servidio@nordicsemi.no>

### CODE REVIEW

**Review by tejlmand — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM

**Comment by FrancescoSer** [comment]
Please find the manifest PR here: https://github.com/nrfconnect/sdk-nrf/pull/3128

**Comment by frkv** [comment]
> Please find the manifest PR here: [nrfconnect/sdk-nrf#3128](https://github.com/nrfconnect/sdk-nrf/pull/3128)

Tnx

File: nrf_security/doc/api.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 API documentation
```
Comment (ru-fu) [inline nrf_security/doc/api.rst:3]: There are more files in this folder that need the ToC.

File: nrf_rpc/doc/architecture.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 Architecture
```
Comment (ru-fu) [inline nrf_rpc/doc/architecture.rst:3]: api.rst in this folder also needs the ToC.

**Review by ru-fu — COMMENTED** [review]
There are a few files missing, see comments.

Also the ZBOSS changelog.

**Review by doki-nordic — APPROVED** [review]

File: nrf_security/doc/api.rst (hunk 1 above)
Comment (FrancescoSer) [reply nrf_security/doc/api.rst:3]: fixed

File: nrf_rpc/doc/architecture.rst (hunk 2 above)
Comment (FrancescoSer) [reply nrf_rpc/doc/architecture.rst:3]: fixed

**Review by ru-fu — APPROVED** [review]

**Review by MarekPorwisz — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**